index.html

```diff
@@ -102,8 +102,15 @@
     <section>
       <h3>Image and Text Profiles</h3>
 
-      <p>While [[!ttml-imsc1.1]] defines both Image and Text Profiles, distinct Image and Text processors SHALL be defined such that
-      an Image Processor is not required to accept Text Profile documents, and vice versa.</p>
+      <p>While [[!ttml-imsc1.1]] defines both Image and Text Profiles, distinct Image and Text processors SHALL be defined such
+      that an Image Processor is not required to accept Text Profile documents, and vice versa.</p>
+    </section>
+
+    <section>
+      <h3>Profile Signaling</h3>
+
+      <p>The [[!ttml-imsc1.1]] profile to which an Instance Document conforms SHALL be identified using the profile signaling
+      mechanisms specified in [[!TTML2]].</p>
```

Comment:
Does this mean that an otherwise conformant EBU-TT-D or IMSC 1 document that does not identify the profile is required to be a non-conformant IMSC 1.1 document? Or does this line allow for the possibility of a content profile signalled by the document interchange context? If that is the intention, then we can not actually impose requirements on the document interchange context since it is by definition out of band.

Reply:
I had not considered this possible interpretation of the requirement.

Could we say the following?

_The [[!ttml-imsc1.1]] profile to which an Instance Document conforms SHALL be identified using the profile signaling mechanisms specified in [[!TTML2]], unless compatibility with IMSC 1.0.1, EBU-TT-D and SDP-US processors is desired._

This matches ttps://github.com/w3c/imsc/pull/267 .

Reply:
Yes, but since that desire is not something that can be mandated or tested, the SHALL doesn't really exist, so we can reword  equivalently to:

_The [[!ttml-imsc1.1]] profile to which an Instance Document conforms SHOULD be identified using the profile signaling mechanisms specified in [[!TTML2]], unless compatibility with IMSC 1.0.1, EBU-TT-D and SDP-US processors is not needed._

Reply:
It can be tested: unless IMSC 1.0.1, EBU-TT-D and SDP-US signaling is used, TTML2 signaling must be used.

Reply:
Ah right, in that case you're not testing the desire (which is a human emotion as far as I know!), you're testing the presence of signalling. 

However there is no requirement for any profile signalling at all in EBU-TT-D, so how does this test work? 

You might say that unless there is a determination from the TTML2 profile resolution mechanism that IMSC 1.0.1, EBU-TT-D or SDP-US applies, then the document needs to include explicit profile signalling, but again, that might depend on the document interchange context, which is out of scope.

I think the best we can do here as part of the requirements is to say that content profile signalling shall be recommended. And leave it at that. Then in the IMSC 1.1 document we say it is a SHOULD and add a note that the explicit signalling of content profile can cause non-conformance with some profiles of TTML, and if such conformance is needed then the non-conformant syntax should not be included.

Reply:
We could say: TTML2 signaling must be used if any TTML2 feature is used,

Reply:
Yes, that would work - assuming you mean "any feature in TTML2 but not in TTML1".

Reply:
> assuming you mean "any feature in TTML2 but not in TTML1".

Yes.